pkg/upgrade/migrate.go

```diff
@@ -333,7 +336,8 @@ func Migrate(ctxt context.Context, c client.Interface, nodename string) error {
 	// calicoDS, err := k8sClient.AppsV1().DaemonSets("kube-system").Get("calico-node", metav1.GetOptions{})
 	// if (calicoDS.Status.UpdatedNumberScheduled == calicoDS.Status.DesiredNumberScheduled-calicoDS.Spec.UpdateStrategy.RollingUpdate.MaxUnavailable.IntVal)
 	log.Info("setting Calico datastore readiness to true...")
-	*clusterInfo.Spec.DatastoreReady = true
+	t := true
+	clusterInfo.Spec.DatastoreReady = &t
```

Comment:
that might be a question of style, or a golang nuance: is there an advantage to doing it this way rather than `*clusterInfo.Spec.DatastoreReady = true` ?

Reply:
I think it's probably fine the way it was... I could probably revert, but I like this better.

I generally like this better because it doesn't dereference a pointer, which might be nil. I think in this case it wouldn't be.. but in other cases we might panic.